### DISCUSSION TITLE
Container stuck in boot loop

### DISCUSSION THREAD

**Discussion by IzMichael** [plausible/analytics#2043] [Self-Hosted Support]
I've installed Plausible with the Docker instructions at [https://plausible.io/docs/self-hosting](https://plausible.io/docs/self-hosting), and I was able to launch it successfully and setup some sites to monitor, but I had to restart my container and now the `plausible_1` container is stuck in a loop.
```
plausible_1            | Loading plausible..
plausible_1            | Starting dependencies..
plausible_1            | Starting repos..
plausible_1            | create Plausible.Repo database if it doesn't exist
plausible_1            | create Plausible.ClickhouseRepo database if it doesn't exist
plausible_1            | Creation of Db successful!
plausible_1            | Loading plausible..
plausible_1            | Starting dependencies..
plausible_1            | Starting repos..
plausible_1            | Running migrations for Elixir.Plausible.Repo
plausible_1            | Running migrations for Elixir.Plausible.ClickhouseRepo
plausible_1            | Migrations successful!
plausible_1            | Loading plausible..
plausible_1            | Starting dependencies..
plausible_1            | Starting repos..
plausible_1            | Admin user already exists. I won't override, bailing
hosting_plausible_1 exited with code 137
```
It just does this ↑ over and over and over again. I'm not able to access the web interface for my installation and restarting the containers / updating them does nothing.
Thanks for any and all help!

**Comment by JacksonChen666** [comment]
a search for [137 exit code](https://duckduckgo.com/?q=137+exit+code&t=ffab&ia=web) contains results with "OOM", indicating that it might be an out of memory issue.
however, if you have lots of available RAM (or available swap space) and that's still happening, it could just be because docker isn't allocating enough resources for the containers.
